ui/src/components/CERN_FOOTER.css

```diff
@@ -152,6 +162,12 @@
   padding-left: 0px;
   float: left;
   width: 66.6666666667%;
+  left: 0;
+  top: -15px;
+}
+
+#cern-footer .footer-third-col {
```

Comment:
I'm very confused, why are you modifying ant's grid? by default it should handle all of these